api/templates/base.html

```diff
@@ -14,7 +14,7 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
 
     <!-- our custom css -->
-    <link rel="stylesheet" href="{% static 'style/custom.css' %}">
+    <link rel="stylesheet" href="{% static 'style/custom.css'  %}?rnd=132">
```

Comment:
what does this do? More for my own interest..

Reply:
While I was changing the .css file the changes wouldn't show on the site immediately so I added this a random version number at the end so it would refresh every time

Reply:
ohh cool, i just keep hitting command+shift+r on my mac to get the stylesheets to reload ha!